ansible/roles/owncloud/defaults/main.yml

```diff
@@ -499,7 +497,7 @@ owncloud__src_remote_dir: '{{
 # .. envvar:: owncloud__apt_repo_source
 #
 # APT ``sources.list`` URL of the ownCloud ``.deb`` repository.
-owncloud__apt_repo_source: '{{ "deb http://" + owncloud__apt_repo_base + "/" +
+owncloud__apt_repo_source: '{{ "deb https://" + owncloud__apt_repo_base + "/" +
```

Comment:
We have ``apt-transport-https``installed right?

Reply:
Not on my clean buster, but the apt version is 1.8.x and can handel https. Should i change to http, for older debian?

Reply:
The `apt` role [will install it by default on older OS releases](https://github.com/debops/debops/blob/master/ansible/roles/apt/defaults/main.yml#L105-L109), on newer ones it is not needed since APT handles `https://` URLs natively. I think you're good here.